.script/tests/KqlvalidationsTests/CustomTables/DSMDataLabelingLogs.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Wondering if there are additional fields that we need to add in the custom table. Might be a good idea to add all the fields instead of just those that might be used in the query.  
![image](https://user-images.githubusercontent.com/37783395/160112495-9de594eb-9b02-4746-808d-7adad464950c.png)
Can we please double check for the two additional custom tables as well to see if additional fields needs to be added

Reply:
Is this the common suggestion?
Because I am afraid that some changes are possible in the near-future, as this is a new table of a new feature.
WDYT?

Reply:
@amramz - It is recommended to include the full schema and we can always update as needed.